### PR TITLE
WIP: Review relaxed fba

### PR DESCRIPTION
**Pull request by marouenbg**
*Please include a short description of enhancement here*
-review of relaxed FBA tutorial
-used load instead of readCbModel recon2.0model.mat 
-found bug in relaxedFBA => error on problem bounds and non minimality of the solution
The relaxation problem assumes from row = 3*+m till row = 4*n+m the following bounds:
lbr = 1 and ubr = max(ub,lb) , where ub and lb are the bounds from the original problem.
Some problems e.g., Recon2 and sIEC have max(lb,ub) < 1 which induces the error.
I turned the lower bound in the relaxation problem to 0, which solved the bounds error.
-tuned gamma parameter as a function of the model size
-wrote l1 approximated relaxedFBA
**I hereby confirm that I have:**

- [x] Tested my code on my own machine
- [x] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [x] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

**Comment by marouenbg** [comment]
@laurentheirendt @syarra Boys, can you put this one on hold, I am trying to debug Recon2 with relaxedFBA. @rmtfleming any suggestion is most welcome.

**Comment by laurentheirendt** [comment]
Is this PR still WIP? 

It would be nice if you could add a quick test (separate PR is fine).

**Comment by marouenbg** [comment]
yes, it needs validation from @rmtfleming

File: tutorials/relaxedFBA/tutorial_relaxedFBA.m [hunk 1]
```diff
@@ -73,21 +73,22 @@
...
-    filename='Recon3.0model';
-    directory='~/work/sbgCloud/programReconstruction/projects/recon2models/data/reconXComparisonModels';
-    model = loadIdentifiedModel(filename,directory);
+    %select your own model, or use Recon2.0model instead
+    if 0
+        filename='Recon3.0model';
+        directory='~/work/sbgCloud/programReconstruction/projects/recon2models/data/reconXComparisonModels';
+        model = loadIdentifiedModel(filename,directory);
+    else
+        filename2='Recon2.0model';
```
Comment (laurentheirendt) [inline tutorials/relaxedFBA/tutorial_relaxedFBA.m]: This should read:
```Matlab
fileName = 'Recon2.0model.mat';
if exist(fileName, 'file') == 2
      model = readCbModel(fileName);
end
```

File: src/analysis/relaxedFBA/relaxFBA_cappedL1.m [hunk 2]
```diff
@@ -319,7 +319,7 @@
     ub2(~excludedReactions) = maxUB;
 
     maxRelaxR = 100;
-    l = [lb2; -maxRelaxR*ones(m,1); zeros(n,1); zeros(n,1); ones(n,1); zeros(m,1)];
+    l = [lb2; -maxRelaxR*ones(m,1); zeros(n,1); zeros(n,1); zeros(n,1); zeros(m,1)];
```
Comment (laurentheirendt) [inline src/analysis/relaxedFBA/relaxFBA_cappedL1.m]: @syarra and @MasoudAhoo, may you check this? Thanks

**Comment by laurentheirendt** [comment]
@marouenbg, may you do a rebase here? Thanks.

**Comment by marouenbg** [comment]
Are there any news about this PR?

**Pull request closed without merging**